ThePensionsRegulator.Frontend/Styles/_tpr-file-upload.scss

```diff
@@ -0,0 +1,36 @@
+﻿.govuk-file-upload {
+    font-size: 1rem;
```

Comment:
Use variables rather than hard-coding. To be the same as `.govuk-button` set body text font size with `@include govuk-font($size: 19, $line-height: 19px);` rather than setting `font-size` directly. (see https://github.com/alphagov/govuk-frontend/blob/main/packages/govuk-frontend/src/govuk/components/button/_index.scss)

Reply:
done